lib/pact_broker/domain/webhook_request.rb

```diff
@@ -56,6 +56,7 @@ class WebhookRequest
       include PactBroker::Logging
       include PactBroker::Messages
       HEADERS_TO_REDACT = [/authorization/i, /token/i]
+      PARAMS_TO_REDACT = [/os_authType/i, /token/i]
```

Comment:
sorry for late replies, have been distracted with 101 other things.

`os_authType` isn't actually a param I need to redact, it was just an example webhook that was included out of the box, and I forgot to remove it.

Co-incidentally neither is `token` haha, I need to redact when it states `circle-token`

aka

`curl -X POST https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/follow?circle-token=:token`

I could code it that if a param contains the word `token` rather than making it circleci specific with `circle-token`

But thinking about this sensibly, it might be better just to have a redactUrl param on the webhook request like you initially suggested.